modules/virt-must-gather-options.adoc

```diff
@@ -33,75 +33,81 @@ Using too many parallel processes can cause performance issues. Increasing the m
 
 .Scripts
 
-Each script is only compatible with certain environment variable combinations.
+Each script is compatible only with certain environment variable combinations.
 
-`gather_vms_details`:: Collect VM log files, VM definitions, and namespaces (and their child objects) that belong to {VirtProductName} resources. If you use this parameter without specifying a namespace or VM, the `must-gather` tool collects this data for all VMs in the cluster. This script is compatible with all environment variables, but you must specify a namespace if you use the `VM` variable.
+`/usr/bin/gather`:: Use the default `must-gather` script, which collects cluster data from all namespaces and includes only basic VM information. This script is compatible only with the `PROS` variable.
 
-`gather`:: Use the default `must-gather` script, which collects cluster data from all namespaces and includes only basic VM information. This script is only compatible with the `PROS` variable.
+`/usr/bin/gather --vms_details`:: Collect VM log files, VM definitions, control-plane logs, and namespaces (and their child objects) that belong to {VirtProductName} resources. If you use this parameter without specifying a namespace or VM, the `must-gather` tool collects this data for all VMs in the cluster. This script is compatible with all environment variables, but you must specify a namespace if you use the `VM` variable.
 
-`gather_images`:: Collect image and image stream custom resource information. This script is only compatible with the `PROS` variable.
+`/usr/bin/gather --images`:: Collect image, image-stream-tags, and image-stream custom resource information. This script is compatible only with the `PROS` variable.
 
 [id="usage-and-examples_{context}"]
 == Usage and examples
 
 Environment variables are optional. You can run a script by itself or with one or more compatible environment variables.
 
+To customize the data that `must-gather` collects, you append a double dash (`--`) to the command, followed by a space and one or more compatible parameters.
+
 [cols="1a,1a", options="header"]
 .Compatible parameters
 |===
 |Script |Compatible environment variable
-|`gather_vms_details`
+|`/usr/bin/gather`   |* `PROS=<number_of_processes>`
+|`/usr/bin/gather --vms_details`
 |* For a namespace: `NS=<namespace_name>`
 
 * For a VM: `VM=<vm_name> NS=<namespace_name>`
 
 * `PROS=<number_of_processes>`
 
-|`gather`   |* `PROS=<number_of_processes>`
-|`gather_images`   |* `PROS=<number_of_processes>`
+
+|`/usr/bin/gather --images`   |* `PROS=<number_of_processes>`
 |===
 
-To customize the data that `must-gather` collects, you append a double dash (`--`) to the command, followed by a space and one or more compatible parameters.
+
 
 .Syntax
```

Comment:
I recommend moving the syntax (rename "Syntax example") so that it appears **before** the table of parameters. That way, the user understands how the parameters fit into the command.

Reply:
Same comment as above; can we address outside of this bug?

Reply:
OK.